types/utils.go

```diff
@@ -36,6 +37,13 @@ func MustSortJSON(toSortJSON []byte) []byte {
 	return js
 }
 
+// Int64ToBigEndian - marshals int64 to a bigendian byte slice so it can be sorted
+func Int64ToBigEndian(i int64) []byte {
```

Comment:
Need to ensure that i >= 0 as well or panic, and add to comments that i cannot be negative.

Reply:
Right, it should really be `Uint64ToBigEndian` then. I should probably change all the ProposalIDs to be uint64 as well.